src/storage/index/b_plus_tree.cpp

```diff
@@ -7,15 +7,39 @@
 #include "storage/page/header_page.h"
 
 namespace bustub {
+
+auto Rp(BufferPoolManager *bpm, page_id_t pid) -> ReadPageGuard {
```

Comment:
We can call buffer pool manager's FetchPageRead / FetchPageWrite / FetchPageBasic instead.

Reply:
Done.